mlflow/pyfunc/__init__.py

```diff
@@ -2400,9 +2416,23 @@ def predict(model_input: List[str]) -> List[str]:
             extra_pip_requirements=extra_pip_requirements,
             model_config=model_config,
             streamable=streamable,
+            model_code_path=model_code_path,
         )
 
 
+# TODO: bbqiu move this to models.utils.py
```

Comment:
will move this after your changes to get rid of backwards compatible langchain stuff @sunishsheth2009

Reply:
Yes we can, lets see if they are 1-1 with configs, etc being added